ledger-service/http-json/src/it/scala/http/HttpServiceIntegrationTest.scala

```diff
@@ -181,20 +273,47 @@ abstract class HttpServiceIntegrationTest
     domain.CreateCommand(templateId, iouT, None)
   }
 
-  private[this] def iouTransfer(
+  private[this] def iouTransfer[Inj](
       locator: domain.ContractLocator[v.Value],
-      to: domain.Party,
+      choice: TExercise[Inj],
+      choiceInterfaceId: Option[domain.ContractTypeId.Interface.OptionalPkg] = None,
```

Comment:
Why `choiceInterfaceId` is not a field of `TExercise` ?

Reply:
No particular reason, take a look at 9757fb02c06 and let me know what you think.